appinfo/routes.php

```diff
@@ -25,7 +25,7 @@
 		//Retrieve Metadata of a list of files
 		[
 			'name' => 'metadata#get_metadata',
-			'url'  => '/metadata/{fileList}',
+			'url'  => '/metadata?fileList={fileList}',
```

Comment:
@oparoz From that doc, I understand that the url should be something like this: `url => '/api/metadata/{fileList}'` but this is not right, I am not able to find what I am missing? 

Also, I am getting an error `[InvalidArgumentException]` when I am trying to run on my system(`vendor/bin/codecept run api`).